src/components/AccountDrawer/index.tsx

```diff
@@ -213,14 +213,14 @@ function AccountDrawer() {
       }
     },
     // reset the yPosition when the user stops dragging
-    onDragEnd: (state) => {
+    onDragEnd: () => {
       setYPosition(0)
       if (scrollRef.current) {
         scrollRef.current.style.overflowY = 'auto'
       }
     },
     // set dragStartTop to true if the user starts dragging from the top of the drawer
-    onDragStart: (state) => {
+    onDragStart: () => {
```

Comment:
nit: this seems unrelated to base, it'd be better to have the linter fixes in their own PR instead of adding them to this one